### ISSUE TITLE
About number of GPUs during training

### ISSUE THREAD

**Issue by phandat128** [Wenjun-Peng/fastformer-for-rec#8]
Hi Wenjun-Peng,
I'm training fastformer for MIND base on your repo. But i don't have enough 4 GPUs, is there any command to set the number of GPUs during training?
Thanks.

**Comment by Wenjun-Peng** [comment]
python train.py \
--pretreained_model unilm \
--pretrained_model_path ./speedymind_ckpts \
--root_data_dir ./data/speedy_data/ \
--news_attributes title \
--num_hidden_layers 8 \
--world_size 1 \
--lr 1e-4 \
--pretrain_lr 8e-6 \
--warmup True \
--schedule_step 240000 \
--warmup_step 1000 \
--batch_size 42 \
--npratio 4 \
--beta_for_cache 0.002 \
--max_step_in_cache 2 \
--savename speedyrec_mind \
--news_dim 256

you should set world size to 1

**Comment by phandat128** [comment]
Thank you for your very fast reply. Could you give me some suggests about batch_size and gradient accumulation when training in fewer GPUs?
Thanks
